demo-pages/pages/demo4.html

```diff
@@ -5,7 +5,11 @@
   <meta name="description" content="An example page for testing" />
 
   <meta type="tab-info" name="enabled" content="true" />
-  <meta type="tab-info" name="info-url" content="http://localhost:3001/api/tabInfo/abcd1234" />
+  <meta
+    type="tab-info"
+    name="info-url"
+    content="https://rdevserver.corp.linkedin.com/v2/rdevmeta/rdev-dfc1d8b5-a1be-4bfc-9b82-995473328b5f"
```

Comment:
We should not check this change in, but it may be useful for testing purposes. Applying the following suggestion in will make it "merge appropriate" 
```suggestion
    content="http://localhost:3001/api/tabInfo/rdev-d50f3b06-5cc1-415a-9b5c-6ed40d06f5fb"
```